doc/pagerank.adoc

```diff
@@ -48,20 +54,54 @@ CREATE (home)-[:LINKS]->(about)
 
 ----
 
-.Running algorithm and streaming results
+==== Running algorithm on a _directed_, _unweighted_ graph and streaming results
+
+.Loading the subset of the graph with labels and relationship type
 [source,cypher]
 ----
 
-CALL algo.pageRank.stream('Page', 'LINKS', {iterations:20, dampingFactor:0.85}) YIELD node, score
+CALL algo.pageRank.stream('Page', 'LINKS', {iterations:20, dampingFactor:0.85}) 
+YIELD node, score
 
 ----
 
-.Running algorithm and writing back results
+.Loading the subset of the graph with cypher query ( can be used to create a virtual graph )
+
 [source,cypher]
 ----
+
+CALL algo.pageRank.stream('MATCH (p:Page) RETURN id(p) as id', 
```

Comment:
should always use LIMIT on the .stream procedures so that people don't accidentally run it on huge graphs.